Gemfile

```diff
@@ -7,15 +7,16 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 4.2.0"
-# This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "legumeinfo-jekyll-theme", "~> 0.0.0"
+#gem "jekyll", "~> 4.2.0"
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
-# gem "github-pages", group: :jekyll_plugins
+gem "github-pages", group: :jekyll_plugins
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
+  gem "jekyll-local-theme", "~> 0.1.1"
+  # need > v1.3.0 to work around https://github.com/avillafiorita/jekyll-datapage_gen/pull/84
+  gem "jekyll-datapage-generator", git: "https://github.com/avillafiorita/jekyll-datapage_gen.git", ref: "23e1299"
```

Comment:
Generating pages for taxa should be a core feature of our theme so this should be in located in the theme's Gemfile isntead.

Is there a reason this is being imported via GitHub instead via RubyGems?